test/parallel/test-stream2-readable-wrap.js

```diff
@@ -1,24 +1,21 @@
 'use strict';
 require('../common');
-var assert = require('assert');
+const assert = require('assert');
+const Readable = require('_stream_readable');
+const Writable = require('_stream_writable');
+const EE = require('events').EventEmitter;
 
-var Readable = require('_stream_readable');
-var Writable = require('_stream_writable');
-var EE = require('events').EventEmitter;
-
-var testRuns = 0, completedRuns = 0;
 function runTest(highWaterMark, objectMode, produce) {
-  testRuns++;
 
-  var old = new EE();
-  var r = new Readable({ highWaterMark: highWaterMark,
+  const old = new EE();
+  const r = new Readable({ highWaterMark: highWaterMark,
                          objectMode: objectMode });
-  assert.equal(r, r.wrap(old));
+  assert.strictEqual(r, r.wrap(old));
 
-  var ended = false;
-  r.on('end', function() {
+  let ended = false;
```

Comment:
This can be removed completely, along with the `assert(ended)` since we're already ensuring 'end' is seen via `common.mustCall()` right below.